package.json

```diff
@@ -41,5 +41,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:3000"
+  "proxy": "http://localhost:5000"
```

Comment:
any reason we moved ports?

Reply:
Chris changed it on the backend side when he made the changes for unified plan